docs/modules/deploy/pages/deploying-on-azure.adoc

```diff
@@ -0,0 +1,309 @@
+        at com.hazelcast.azure.RestClient.get(RestClient.java:69)
+        at com.hazelcast.azure.AzureMetadataApi.callGet(AzureMetadataApi.java:107)
+        at com.hazelcast.azure.AzureMetadataApi.accessToken(AzureMetadataApi.java:96)
+        at com.hazelcast.azure.AzureClient.fetchAccessToken(AzureClient.java:131)
+        at com.hazelcast.azure.AzureClient.getAddresses(AzureClient.java:118)
+        at com.hazelcast.azure.AzureDiscoveryStrategy.discoverNodes(AzureDiscoveryStrategy.java:136)
+        at com.hazelcast.spi.discovery.impl.DefaultDiscoveryService.discoverNodes(DefaultDiscoveryService.java:71)
+        at com.hazelcast.internal.cluster.impl.DiscoveryJoiner.getPossibleAddresses(DiscoveryJoiner.java:69)
+        at com.hazelcast.internal.cluster.impl.DiscoveryJoiner.getPossibleAddressesForInitialJoin(DiscoveryJoiner.java:58)
+        at com.hazelcast.internal.cluster.impl.TcpIpJoiner.joinViaPossibleMembers(TcpIpJoiner.java:136)
+        at com.hazelcast.internal.cluster.impl.TcpIpJoiner.doJoin(TcpIpJoiner.java:96)
+        at com.hazelcast.internal.cluster.impl.AbstractJoiner.join(AbstractJoiner.java:137)
+        at com.hazelcast.instance.impl.Node.join(Node.java:796)
+        at com.hazelcast.instance.impl.Node.start(Node.java:451)
+        at com.hazelcast.instance.impl.HazelcastInstanceImpl.<init>(HazelcastInstanceImpl.java:122)
+        at com.hazelcast.instance.impl.HazelcastInstanceFactory.constructHazelcastInstance(HazelcastInstanceFactory.java:241)
+        at com.hazelcast.instance.impl.HazelcastInstanceFactory.newHazelcastInstance(HazelcastInstanceFactory.java:220)
+        at com.hazelcast.instance.impl.HazelcastInstanceFactory.newHazelcastInstance(HazelcastInstanceFactory.java:158)
+        at com.hazelcast.core.Hazelcast.newHazelcastInstance(Hazelcast.java:91)
+        at com.hazelcast.core.server.HazelcastMemberStarter.main(HazelcastMemberStarter.java:46)
+Caused by: com.hazelcast.azure.RestClientException: Failure executing: GET at: http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https://management.azure.com. Message: {"error":"invalid_request","error_description":"Identity not found"},
+        at com.hazelcast.azure.RestClient.call(RestClient.java:101)
+        ... 20 more
+```
+
+== Discovering Members from Hazelcast Clients
+ 
+If you have Hazelcast clients running outside of an Azure VM, the Azure Instance Metadata service unavailable. In this case, Hazelcast client instances should be configured with the following properties:
+
+NOTE: You will need to setup link:https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/[Azure Active Directory Service Principal credentials] for your Azure Subscription to be able to use these properties.
+
+- `instance-metadata-available` - This property should be configured as `false` in order to be able to use the following properties. It is `true` by default.
```

Comment:
For property descriptions, I see (in this plugins documentation) we also use ":" and then start with non-capitalized wording, e.g., 

`instance-metadata-available`: should be configured as ......

Maybe it is better to standardize this kind of content (it can be a TBD for future content improvements)